### PR TITLE
Feature/review changes

### PR DESCRIPTION
**Pull request by drobnikj**

### CODE REVIEW

File: src/creates/actor_run.js [hunk 1]
```diff
@@ -21,24 +21,29 @@ const runActor = async (z, bundle) => {
...
-
-    let run = runResponse.json;
-    if (runSync) {
-        run = await enrichActorRun(z, run);
+    if (inputBody) {
+        if (inputContentType.includes('application/json')) {
+            try {
+                JSON.parse(inputBody);
+            } catch (err) {
+                throw new Error('Please check that your input body is valid JSON.');
```
Comment (jancurn) [inline src/creates/actor_run.js]: is **a** valid JSON

File: src/creates/set_value.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ const setValue = async (z, bundle) => {
     try {
         valueObject = JSON.parse(value);
     } catch (err) {
-        throw new Error(`Cannot parse the value as JSON: ${err.message}`);
+        throw new Error('Please check that your record value is valid JSON.');
```
Comment (jancurn) [inline src/creates/set_value.js]: is **a** valid JSON

File: src/creates/actor_run.js (hunk 1 above)
Comment (mtrunkat) [reply src/creates/actor_run.js]: Shouldn't we say "Your input is not a valid JSON: [errorMsg]" as we can be sure at this point that it's not valid?

Comment (drobnikj) [reply src/creates/actor_run.js]: The Zapier team wants to make error messages as simple as possible to BFU.

**Pull request merged**